microservice/frontend/catalog/debian/data/etc/nginx/sites-available/shop.microservice.io.1

```diff
@@ -0,0 +1,14 @@
+server {
+  listen   81; ## listen for ipv4; this line is default and implied
+  #listen   [::]:80 default ipv6only=on; ## listen for ipv6
+
+  root /usr/share/shop_A/frontend/catalog/html;
```

Comment:
hi @ojacques no sure to understand this comment, if I am understanding correctly we should be using different path? . . . also tried to deploy changes on vlab ending with 11 but jenkins is sending an error saying that it cannot lock a file, trying to search a solution but no luck . . . commit my latest changes here, if you go to the microservice pro box you will find there my changes too and will see that those are working (without the site_A and site_B) but could no test it properly running jenkins